pallets/asset-handler/src/tests.rs

```diff
@@ -587,6 +587,64 @@ pub fn test_convert_amount_for_foreign_chain() {
 	});
 }
 
```

Comment:
Add a case where the recipient account doesn't exist and Thea mints into it. or burns. Let's see the error handling

Reply:
Done ✅